lib/line/bot/client.rb

```diff
@@ -333,15 +333,18 @@ def get_room_member_profile(room_id, user_id)
 
       # Get user IDs of who added your LINE Official Account as a friend
       #
-      # @param continuation_token [String] Identifier to return next page
-      #                                   (next property to be included in the response)
+      # @param start [String] Identifier to return next page (next property to be included in the response)
+      # @param limit [Integer] The maximum number of user IDs to retrieve in a single request
       #
       # @return [Net::HTTPResponse]
-      def get_follower_ids(continuation_token = nil)
+      def get_follower_ids(start: nil, limit: nil)
         channel_token_required
 
+        params = { start: start, limit: limit }.compact
+
         endpoint_path = "/bot/followers/ids"
-        endpoint_path += "?start=#{continuation_token}" if continuation_token
+        endpoint_path += "?" unless params.empty?
+        endpoint_path += params.map { |key, value| "#{key}=#{value}" }.join("&")
```

Comment:
How about using this method?
```
URI.encode_www_form
```
https://github.com/line/line-bot-sdk-ruby/blob/d53025428765b069770e6d12ca3ba448f59b511f/lib/line/bot/client.rb#L94-L98

Reply:
Thank you! I fixed it using that method.
https://github.com/line/line-bot-sdk-ruby/pull/246/commits/c5bc1739fe9891d4ea33893cbf5a5fa052f9288f